content/en/docs/components/katib/getting-started.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "Getting Started"
+description = "Get started with Katib"
+weight = 30
++++
+
+This guide describes how to get started with Katib and run a few examples.
+
+## Prerequisites
+
+You need to install the following Katib components to run examples:
+
+- Katib [installed](/docs/components/katib/installation/#installing-katib).
+- Katib Python SDK [installed](/docs/components/katib/installation/#installing-katib-python-sdk).
+
+## Getting Started with Katib Python SDK
```

Comment:
I'm wondering if we can the guide about `## Getting Started with kubectl` here as well.
If we can add that, I think that we can move https://github.com/kubeflow/katib/tree/1551ca3975ed50a5af3946bb8847e908888ce27e/examples/v1beta1/kind-cluster to here.

@andreyvelich WDYT?

Reply:
That is interesting, but I already added the section about [getting started with Experiment YAML.](https://deploy-preview-3723--competent-brattain-de2d6d.netlify.app/docs/components/katib/getting-started/#getting-started-with-katib-experiment-yaml)
For me the `kind` guides relate to Developer guides section since [prerequisite for Katib getting started](https://deploy-preview-3723--competent-brattain-de2d6d.netlify.app/docs/components/katib/getting-started/#prerequisites) is to install Katib Control Plane. 
Eventually, we want to split our docs like this:
- User Guides (how to run Katib Experiments)
- Administrators Guides (how to install and configure Katib ControlPlane)
- Developer Guides (how to contribute and develop Katib platform)

Reply:
> That is interesting, but I already added the section about [getting started with Experiment YAML.](https://deploy-preview-3723--competent-brattain-de2d6d.netlify.app/docs/components/katib/getting-started/#getting-started-with-katib-experiment-yaml)

Oh, I was missing the section. So, I'm ok with keeping the current getting started. Thanks!